File: LeetCode_3190/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int minimumOperations(vector<int>& nums) {
        int cnt = 0;
        for (int num : nums) {
            if (num % 3) {
                cnt += 1;
            }
        }
        return cnt;
    }
};

File: LeetCode_3190/Main.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function(nums) {
    var cnt = 0;
    for (let num of nums) {
        if (num % 3) {
            cnt += 1;
        }
    }
    return cnt;
};